### PR TITLE
Resolve issue with avatars on docs/guide page

### PR DESCRIPTION
**Pull request by daveoconnor**
Related to #1307, couldn't reproduce that specific issue, but did see an issue with the avatar sizing and styling, so added tailwind to docs template

### CODE REVIEW

File: templates/docs_libs_placeholder.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {% load static %}
 
 {% block extra_head %}
+  <link href="{% static 'css/styles.css' %}" rel="stylesheet" />
```
Comment (vinniefalco) [inline templates/docs_libs_placeholder.html:5]: @rbbeeston does the avatar partial require tailwind?

**Comment by daveoconnor** [comment]
PR updated with fix for avatar status being lost on cached pages. This is related to the removal of HTMX loading from this PR.

https://github.com/boostorg/website-v2/pull/1298

File: templates/docs_libs_placeholder.html (hunk 1 above)
Comment (rbbeeston) [reply templates/docs_libs_placeholder.html:5]: The avatar partials currently use tailwind, yes. We can convert it as another ticket.

**Review by rbbeeston — APPROVED** [review]
LGTM!

**Pull request merged**